var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -170,6 +170,20 @@ def make_defs(self):
 
         return make_defs
 
+    @property
+    def headers(self):
+        include_dir = self.spec.prefix.include
+
+        # As in netlib-lapack, the only public headers for cblas and lapacke in
+        # openblas are cblas.h and lapacke.h. The remaining headers are private
+        # headers either included in one of these two headers, or included in
+        # one of the source files implementing functions declared in these
+        # headers.
+        header_basenames = ['cblas.h', 'lapacke.h']
+        header_pathnames = [join_path(include_dir, h)
+                            for h in header_basenames]
+        return HeaderList(header_pathnames)
```

Comment:
You can replace this entire thing with:
```suggestion
return find_headers(['cblas', 'lapacke'], self.prefix.include)
```

Reply:
Done, plus removed an unused variable, and rebased onto current `develop`.